utils.pwsh/Setup-Target.ps1

```diff
@@ -43,6 +43,44 @@ function Setup-BuildParameters {
         ($VisualStudioData.DisplayName -split ' ')[-1]
     )
 
+    $script:CFlags = @()
+    $script:CxxFlags = @()
+
+    switch ( ${script:Configuration} ) {
+        Debug {
+            $script:CFlags += @(
+                '-Ob0 -Od -RTC1'
+            )
+            $script:CxxFlags += @(
+                '-Ob0 -Od -RTC1'
+            )
+        }
+        RelWithDebInfo {
+            $script:CFlags += @(
+                '-O2 -Ob1 -DNDEBUG'
+            )
+            $script:CxxFlags += @(
+                '-O2 -Ob1 -DNDEBUG'
+            )
+        }
+        Release {
+            $script:CFlags += @(
+                '-O2 -Ob2 -DNDEBUG'
+            )
+            $script:CxxFlags += @(
+                '-O2 -Ob2 -DNDEBUG'
+            )
+        }
+        MinSizeRel {
+            $script:CFlags += @(
+                '-O1 -Ob1 -DNDEBUG'
+            )
+            $script:CxxFlags += @(
+                '-O1 -Ob1 -DNDEBUG'
+            )
+        }
+    }
```

Comment:
Some project overwrite the CMake defaults with their own (bad) variations iirc, this enforces the defaults again. Annoying but necessary because some projects seem to have converted their autotools-based systems straight to CMake without regard for its defaults.

Reply:
Unfortunate. I don't like having to compensate and maintain stuff like this in perpetuity, but I'm not sure it's easily avoidable here. We should attempt to see where we can alleviate this in our dependencies.